tests/conformance_tests/src/lib.rs

```diff
@@ -12,7 +12,7 @@ pub use self::message::*;
 pub use self::rand_replay::*;
 pub use self::stubs::*;
 pub use self::tipset::*;
-use actor::CHAOS_ACTOR_CODE_ID;
+use actor::actorv2::CHAOS_ACTOR_CODE_ID;
```

Comment:
Curious how you're passing v1 conformance tests with v2 chaos actor

Reply:
The code Cid is still using the /1/ path. We probably shouldn't even have chaos within actors because it isn't tied to the actor versions. But can change later